Makefile

```diff
@@ -216,6 +218,9 @@ tests_console = $(sort $(patsubst tests/%.console_out,%,$(wildcard tests/*.conso
 
 tests_console: $(tests_console)
 
+check_vunit:
+	$(VUNITRUN) -p10
```

Comment:
I suggest `$(VUNITRUN) $(VUNITARGS)` and (above) `VUNITARGS ?= -p10`.

The `run.py` entrypoint supports CLI arguments such as the following:

- `-v`: verbose
- `-g`: open GUI
- `lib.testbench_*.*`: wildcards for filtering which tb/tests to run
- `x`: generate an XUnit report (XML)

See [vunit.github.io/cli](http://vunit.github.io/cli.html).

Reply:
Thanks @umarcor I've updated and repushed.